OpenSim/Simulation/Model/Force.h

```diff
@@ -47,14 +47,18 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Force, ModelComponent);
 //==============================================================================
 // PROPERTIES
 //==============================================================================
-    /** A Force element is active (enabled) by default. **/
-    OpenSim_DECLARE_PROPERTY(isDisabled, bool,
-        "Flag indicating whether the force is disabled or not. Disabled means"
+    /** A Force element is active (enabled) by default.
+    NOTE: Prior to OpenSim 4.0, this property was named **isDisabled**.
+          If **isDisabled** is **true**, **appliesForce** is **false**.
+          If **isDisabled** is **false**, **appliesForce** is **true**.          */
+    OpenSim_DECLARE_PROPERTY(appliesForce, bool,
+        "Flag indicating whether the force is applied or not. Value of false means"
         " that the force is not active in subsequent dynamics realizations.");
```

Comment:
I looked at the generated Doxygen. It looks fine.

I have updated the property comment to mention the change in property name.